mod/gcforums/views/default/alerts/delete.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+$entity = $vars['entity'];
+$is_menu_buttons = $vars['is_menu_buttons'];
+$delete_title = elgg_echo('gcforums:delete:heading');
+$delete_content = elgg_echo('gcforums:delete:body', array(elgg_echo("gcforums:translate:{$entity->getSubtype()}")));
+$delete_button = elgg_echo('gcforums:delete:delete');
+$cancel_button = elgg_echo('gcforums:delete:cancel');
+
+$url = elgg_add_action_tokens_to_url(elgg_get_site_url()."action/gcforums/delete?guid={$entity->getGUID()}");
+$delete = "<a style='color:white; text-decoration:none;' href='{$url}'>{$delete_button}</a>";
+$button_size = ($is_menu_buttons) ? "" : "btn-sm";
+
+?>
+
+<button type="button" class="btn btn-danger <?php echo $button_size; ?>" data-toggle="modal" data-target="#myModal-<?php echo $entity->guid; ?>"><?php echo $delete_button; ?></button>
+
+<div id="myModal-<?php echo $entity->guid; ?>" class="modal fade" role="dialog">
+	<div class="modal-dialog">
+
+		<div class="modal-content">
+			<div class="modal-header">
+		    	<button type="button" class="close" data-dismiss="modal">&times;</button>
+		    	<h4 class="modal-title"><?php echo $delete_title; ?></h4>
+		    </div>
+		  	<div class="modal-body">
+			    <p><?php echo $delete_content . " '{$entity->title}'"; ?></p>
+		    </div>
+			<div class="modal-footer">
+				<button id="btnDeleteEntity" type="button" class="btn btn-danger <?php echo $button_size; ?>" data-toggle="modal" data-target="#myModal"><?php echo $delete; ?></button>
```

Comment:
On preprod this doesn't seem to be deleting the forum. I believe it's because the $delete link is inside the button. If you style $delete with the classes "btn btn-danger" it might be able to replace this button element.

Reply:
delete modal looks good 👌

Reply:
the delete is working, i the action file was checking the wrong condition